packages/framework/src/Resources/styles/admin/main.less

```diff
@@ -43,6 +43,9 @@
 @import "libs/tooltip/tooltip-custom.less";
 @import "libs/animation.less";
 
+@import "libs/jquery/theme.less";
+@import "libs/jquery/datepicker.less";
```

Comment:
Does it make sense to do a code review from frontend-developers for introduced less files? 
TBH I'm not sure if all the rules are necessary or not.

Reply:
I consulted this amendment with mr. Gott. What do you thing with `all the rules`?

Reply:
CSS selectors and declarations in `theme.less` and `datepicker.less`. 
My question is mainly _"Do we need/want the whole files like they are?"_

Reply:
Resolved personally, files are ok